sirepo/simulation_db.py

```diff
@@ -904,14 +892,13 @@ def _init():
         sbatch_display=(None, str, 'how to display sbatch cluster to user'),
         tmp_dir=(None, pkio.py_path, 'Used by utilities (not regular config)'),
     )
-    fn = STATIC_FOLDER.join('json/schema-common{}'.format(JSON_SUFFIX))
-    with open(str(fn)) as f:
+    with open(str(sirepo.resource.static('json', f'schema-common{JSON_SUFFIX}'))) as f:
         SCHEMA_COMMON = json_load(f)
     # In development, any schema update creates a new version
     if pkconfig.channel_in('dev'):
         SCHEMA_COMMON.version = max([
-            _timestamp(pkio.py_path(fn).mtime()) \
-            for fn in glob.glob(str(STATIC_FOLDER.join('json/*{}'.format(JSON_SUFFIX))))
+            _timestamp(fn.mtime()) \
+            for fn in sirepo.resource.glob_static(f'json/*{JSON_SUFFIX}')
```

Comment:
As long as we are abstracting in this way, make the interfaces parallel: glob_static('json'). There's no need to expose more of the interface.

Reply:
eb1b916d